brayns/engine/volume/Volume.cpp

```diff
@@ -33,6 +48,7 @@ Volume &Volume::operator=(const Volume &other)
     _volumeName = other._volumeName;
     _handle = ospray::cpp::Volume(_handleName);
     _data = other._data->clone();
+    _flag = true;
```

Comment:
This looks a bit weird, I would do something like _flag.setModified(true).

Reply:
Will change on next PR